cfy-go/instances.go

```diff
@@ -217,39 +221,41 @@ func nodeInstancesOptions(args, options []string) int {
 	case "by-type":
 		{
 			operFlagSet := basicOptions("node-instances started")
-			var nodeType string
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesNode, "Filter by node type")
-
 			params := parseInstancesFlags(operFlagSet, options)
 
+			var nodeType = os.Getenv("CFY_K8S_NODE_TYPE")
+			if nodeType == "" {
+				nodeType = cloudify.KubernetesNode
+			}
+
 			cl := getClient()
 			return nodeInstancesPrint(
 				cl.GetNodeInstancesWithType(params, nodeType))
 		}
 	case "started":
 		{
 			operFlagSet := basicOptions("node-instances started")
-			var nodeType string
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesNode, "Filter by node type")
-
 			params := parseInstancesFlags(operFlagSet, options)
 
+			var nodeType = os.Getenv("CFY_K8S_NODE_TYPE")
+			if nodeType == "" {
+				nodeType = cloudify.KubernetesNode
+			}
+
 			cl := getClient()
 			return nodeInstancesPrint(
 				cl.GetStartedNodeInstancesWithType(params, nodeType))
 		}
 	case "alive":
 		{
 			operFlagSet := basicOptions("node-instances alive")
-			var nodeType string
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesNode,
-				"Filter by node type: cloudify.nodes.ApplicationServer.kubernetes.{Node|LoadBalancer}")
-
```

Comment:
Use: 
```
operFlagSet.StringVar(&nodeType, "node-type",
				os.Getenv("CFY_K8S_NODE_TYPE"), "Filter by node type")
```

Reply:
@0lvin check https://github.com/cloudify-incubator/cloudify-rest-go-client/tree/testing/cfy-go module it is up to date

Reply:
I does not see param for provide load balancer type, please restore such:

* https://github.com/cloudify-incubator/cloudify-rest-go-client/blob/f1173dcd06e1dd31663bf124d494ce0c856eac21/cfy-go/instances.go#L210
* https://github.com/cloudify-incubator/cloudify-rest-go-client/blob/f1173dcd06e1dd31663bf124d494ce0c856eac21/cfy-go/instances.go#L226
* https://github.com/cloudify-incubator/cloudify-rest-go-client/blob/f1173dcd06e1dd31663bf124d494ce0c856eac21/cfy-go/instances.go#L240